matching-game/src/components/variable-text/variable-text.module.css

```diff
@@ -0,0 +1,6 @@
+.variable {
+    font-family: 'Radio Canada', sans-serif;
+    color: #FFFFFF;
```

Comment:
we'll consolidate some of this later, since this will be our standard font info. could make sense in the globals, but it's not super necessary.